graphql/resolvers.go

```diff
@@ -899,7 +899,7 @@ func (r *taskResolver) SpawnHostLink(ctx context.Context, at *restModel.APITask)
 	}
 	hasSpawnableProvider := false
 	for _, spawnableProvider := range evergreen.ProviderSpawnable {
```

Comment:
why not have a global list of user-spawnable provider and then use util.StringSliceContains() rather than writing the loop and having the special case here.